common/src/main/java/com/wynntils/models/abilities/AbilityTreeModel.java

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.abilities;
+
+import com.wynntils.core.components.Model;
+import java.util.List;
+
+public class AbilityTreeModel extends Model {
```

Comment:
I have decided to move all parsing to `ParsedAbilityTree`. This will make it so this model is not cluttered, and mostly focuses on runtime manipulation of ability trees, while parsing will reside in the parser class.